src/components/tools/drawing-tool/drawing-tool.sass

```diff
@@ -4,8 +4,11 @@
   height: 100%
 
 .drawing-tool-toolbar
+  position: absolute
+  height: 44px
```

Comment:
Do the toolbars all have a set height that should be stored in a constant?  Or do the heights vary depending on content and type?

Reply:
Right now I'm making the toolbars float without updating their appearance, so they all have different dimensions, etc. Once we've standardized on Michael's designs there will probably be some sharable constants.